.github/workflows/ci-desktop-app.yml

```diff
@@ -1,4 +1,4 @@
-name: Desktop App Build
+name: Desktop App Build (without signing)
```

Comment:
meganit: slightly prefer to put this just in ci file since it can handle any number of jobs?